galaxy.yml

```diff
@@ -15,7 +15,7 @@ tags:
   - satellite
   - orcharhino
```

Comment:
in `redhat.satellite` we had removed that tag

Reply:
I will address this on next push.